showcase/radar-chart/animated-radar-chart.js

```diff
@@ -72,6 +72,9 @@ export default class AnimatedRadar extends Component {
               text: {
                 opacity: 0
               }
+            },
+            labels: {
```

Comment:
It's just for showing the effect, or the default `labels` style will be override by an empty `{}`.